rest/route/test_routes.go

```diff
@@ -64,9 +65,18 @@ func (tgh *testGetHandler) ParseAndValidate(ctx context.Context, r *http.Request
 			StatusCode: http.StatusNotFound,
 		}
 	}
+	execution := r.URL.Query().Get("execution")
+	executionInt, err := strconv.Atoi(execution)
+	if err != nil && execution != "" {
```

Comment:
Just to confirm, you want the query parameter to be mandatory right?

Reply:
I find this a little hard to read tbh. Personally I do something like this:
```
var executionInt int
execution := r.URL.Query().Get("execution")
if execution != "" {
executionInt, err = strconv.Atoi(execution)
...
}
```

Reply:
fixed